website/views/user.py

```
@@ -1011,14 +1014,275 @@ def github_webhook(request):
+            number=pr_number,
+            defaults={
+                "title": pr_data.get("title", ""),
+                "type": "pull_request",
+                "user_profile": pr_user_profile,
+                "is_merged": True,
+                "merged_at": timezone.now(),
+            },
+        )
+
+        # If issue exists but missing user_profile, set it
+        if not created and github_issue.user_profile is None:
```

Ruff (0.14.5)

Undefined name `created`

(F821)